packages/web/src/common/store/cache/collections/sagas.js

```diff
@@ -869,10 +869,21 @@ function* publishPlaylistAsync(action) {
     ])
   )
 
-  yield call(confirmPublishPlaylist, userId, action.playlistId, playlist)
+  yield call(
+    confirmPublishPlaylist,
+    userId,
+    action.playlistId,
+    playlist,
+    action.confirmCallback
```

Comment:
so maybe here we have the toastKey or something?

Reply:
another option is to allow us to pass a toast key to the toast dispatch, so we can simply reference it anywhere else in the app when we want to dismiss.